### PR TITLE
Added auto push pending changes

### PR DESCRIPTION
**Pull request by SeqLaz**
Added section of the new functionality of Automatically push pending changes on the following interval

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3896 #566 Allow users to force auto-push and manage time interval via project configuration](https://app.clickup.com/t/2192114/QF-3896)

**Comment by nirvn** [comment]
@SeqLaz , nice, quite an important functionality to highlight, thanks for being so quick! Could we use this screenshot (taken from a mobile device) instead:



It has slightly better proportions and the generic phone frame isn't pixelated. On that, I'd be happy to have a quick chat with you and explain how to achieve the "rounded screen corner" effect on inkscape.

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 1]
```diff
@@ -264,6 +264,29 @@ Any QField supported raster and vector layer formats may be used as read-only da
 
 !![Project properties](../../assets/images/getting_started_project_properties.png)
 
+### Enabling Auto-Push and Setting Push Interval in QFieldCloud
+
+With this functionality, users or managers of QFieldCloud projects can enforce auto-push on QField devices in the field and specify the interval in minutes, optimizing field data management processes.
```
Comment (nirvn) [inline documentation/get-started/tutorials/get-started-qfc.en.md:269]:
```suggestion
With this functionality, users and managers of QFieldCloud projects can enforce automatic pushing of pending changes for QField devices in the field as well as specify the interval in between automated pushes. The functionality is activated through a project setting, allowing for the remote activation.
```

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 2]
```diff
@@ -264,6 +264,29 @@ Any QField supported raster and vector layer formats may be used as read-only da
...
+
+1. **Access Project Settings**: Navigate to the settings in QFiedlSync.
```
Comment (nirvn) [inline documentation/get-started/tutorials/get-started-qfc.en.md:271]:
```suggestion
1. **Access Project Settings**: Navigate to the QField panel in the Project Settings dialog provided by the QFieldSync plugin.
```

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 3]
```diff
@@ -264,6 +264,29 @@ Any QField supported raster and vector layer formats may be used as read-only da
...
+
+2. **Enable Auto-Push**: Toggle the "Automatically push pending changes on the following interval" option and establish your prefer interval.
```
Comment (nirvn) [inline documentation/get-started/tutorials/get-started-qfc.en.md:273]:
```suggestion
2. **Enable Auto-Push**: Toggle the "Automatically push pending changes on the following interval" option and establish your preferred interval.
```

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 4]
```diff
@@ -264,6 +264,29 @@ Any QField supported raster and vector layer formats may be used as read-only da
...
+
+!![Auto push QFieldSync](../../assets/images/auto-push-pending-changes-qfieldsync.png)
+
+!![Auto push QField](../../assets/images/auto-push-pending-changes-qfield.png,400px)
+
+!!! note
+    #### Benefits:
+
+    - **Real-Time Updates**: Ensures prompt synchronization of field data with the QFieldCloud project.
+    - **Streamlined Workflow**: Minimizes manual intervention and ensures surveyors not worry in the synchronization and only on the data quality.
```
Comment (nirvn) [inline documentation/get-started/tutorials/get-started-qfc.en.md:283]:
```suggestion
    - **Streamlined Workflow**: Minimizes manual intervention and ensures surveyors do not need to worry about synchronization, helping them focus on data quality.
```

**Comment by SeqLaz** [comment]
Really thank you @nirvn, that would be great!

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 5]
```diff
@@ -264,6 +264,29 @@ Any QField supported raster and vector layer formats may be used as read-only da
 
 !![Project properties](../../assets/images/getting_started_project_properties.png)
 
+### Enabling Automatic Pushing of Changes to QFielcCloud
```
Comment (lucienicolier) [inline documentation/get-started/tutorials/get-started-qfc.en.md:267]:
```suggestion
### Enabling Automatic Pushing of Changes to QFieldCloud
```

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 6]
```diff
@@ -264,6 +264,29 @@ Any QField supported raster and vector layer formats may be used as read-only da
...
+
+With this functionality, users and managers of QFieldCloud projects can enforce automatic pushing of pending changes for QField devices in the field as well as specify the interval in between automated pushes. The functionality is activated through a project setting, allowing for the remote activation.
```
Comment (lucienicolier) [inline documentation/get-started/tutorials/get-started-qfc.en.md:269]:
```suggestion
With this functionality, users and managers of QFieldCloud projects can enforce automatic pushing of pending changes to QField devices in the field, as well as specify the interval in between automated pushes. The functionality is activated through a project setting, allowing remote activation.
```

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 7]
```diff
@@ -264,6 +264,29 @@ Any QField supported raster and vector layer formats may be used as read-only da
...
+
+1. **Access Project Settings**: Navigate to the QField panel in the Project Settings dialog provided by the QFieldSync plugin.
+
+2. **Enable Auto-Push**: Toggle the "Automatically push pending changes on the following interval" option and establish your preferred interval.
+
+!![Auto push QFieldSync](../../assets/images/auto-push-pending-changes-qfieldsync.png)
+
+!![Auto push QField](../../assets/images/auto-push-pending-changes-qfield.png,400px)
+
+!!! note
```
Comment (lucienicolier) [inline documentation/get-started/tutorials/get-started-qfc.en.md:279]:
```suggestion
!!! Note
```

**Pull request merged**